.eslintrc.js

```diff
@@ -1,15 +1,78 @@
 module.exports = {
-  root: true,
-  env: {
-    browser: true,
-    node: true,
-  },
-  extends: [
-    '@nuxtjs/eslint-config-typescript',
-    'plugin:prettier/recommended',
-    'plugin:nuxt/recommended',
-  ],
-  plugins: [],
-  // add your custom rules here
-  rules: {},
-}
+    root: true,
+    env: {
+        browser: true,
+        node: true
+    },
+    extends: [
+        "@nuxtjs/eslint-config-typescript",
+        // 'plugin:prettier/recommended',
+        "plugin:nuxt/recommended"
+    ],
+    plugins: [],
+    rules: {
+        semi: [ "warn", "always" ],
+        quotes: [ "warn", "double", { allowTemplateLiterals: true } ],
+        "max-len": [
+            "warn",
+            {
+                code: 180,
+                tabWidth: 4,
+                ignoreUrls: true
+            }
+        ],
+        indent: [
+            "warn",
+            4,
+            {
+                SwitchCase: 1
+            }
+        ],
+        "vue/max-attributes-per-line": [
+            "warn",
+            {
+                singleline: 1,
```

Comment:
Come numero di attributi singleline proporrei di aumentare a 2, ci sono situazioni in cui averne due aiuta a snellire il codice senza comprometterne la lettura, e.g. i v-for

```suggestion
                singleline: 2,
```

Reply:
Generalmente non credo avremo componenti con tantissime prop quindi probabilmente 1 va bene, possiamo cambiarlo tranquillamente se vediamo che diventa troppo verticale.